chart/values.schema.json

```diff
@@ -5254,6 +5254,11 @@
                             "default": "tests/dags"
                         },
                         "wait": {
+                            "description": "Interval between git sync attempts in seconds. High values are more likely to cause DAGs to become out of sync between different components. Low values cause more traffic to the remote git repository. (Deprecated. Use 'period')",
+                            "type": "integer",
+                            "default": 5
```

Comment:
@jedcunningham We chatted a bit about potentially changing this value. Is that something we want to ride in on this PR?

Reply:
We [just reduced this from 60](https://github.com/apache/airflow/pull/27625) to reduce the risk of them being out of sync. If you want to pitch another value, definitely so that in another PR.